helm/pinot/values.yaml

```diff
@@ -92,6 +96,21 @@ controller:
     endpoint: "/health"
     livenessEnabled: false
     readinessEnabled: false
+    startupEnabled: false
+    liveness:
+      initialDelaySeconds:
```

Comment:
is this valid yaml? shall we put values here?

Reply:
Yes, it's a valid one. Here is the output
```bash
helm lint                                                                                              
==> Linting .
[INFO] Chart.yaml: icon is recommended

1 chart(s) linted, 0 chart(s) failed
```